### DISCUSSION TITLE
GitHub pages site not appearing in any search engine

### DISCUSSION THREAD

**Discussion by nicolaevansfineart** [community/community#23074] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I’ve created a site which is hosted on GitHub pages which isn’t appearing on any search engine.</p>
<p>The site is <a href="https://nicolaevansfineart.com/" rel="nofollow noopener">https://nicolaevansfineart.com/</a></p>
<p>It works just by going to the URL and it has been successfully crawled and indexed by Google but it still isn’t showing.</p>
<p>Can anyone help? I am at the end of my rope here!</p>

**Comment by tajmone** [comment, marked as answer]
<p>It won’t show up immediately on Google due to its sandbox feature — i.e. a test period in which Google want’s to ensure that a new website is not a search-engine trap. i.e. that as soon as it’s index it will change all its contents. We don’t know much about the details of how Google handles the whole process, but I remember when the sandbox feature first came out (a couple of decades ago) to counter the above mentioned practice (e.g. waiting for a site to be indexed for some given keywords, and then switch to different contents, often counterproductive to the keywords for which it appeared).</p>
<p>Before the sandbox era, you could pretty much count on your website showing up at specific dates (beginning of upcoming month) once it was crawled. Now it’s a matter of waiting, and only those at Google know how it works and how long you need to wait.</p>
<p>Usually, a couple of weeks should be enough for a new website to show up in search engines, a month at the most.</p>

**Comment by tajmone** [comment]
<p>In any case, if you search <code>nicolaevansfineart.com</code> on Google your GHPages already shows up.</p>

**Comment by nicolaevansfineart** [comment]
<p>Thanks for that info <a class="mention" href="/tajmone">@tajmone</a> I had no idea about the sandboxing stuff!</p>
<p>I think it started showing up over night after about a week of waiting!</p>
<p>Anyway thanks for your help :grinning:</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="nicolaevansfineart" data-post="4" data-topic="121028">
<div class="title">
<div class="quote-controls"></div>
 nicolaevansfineart:</div>
<blockquote>
<p>I think it started showing up over night after about a week of waiting!</p>
</blockquote>
</aside>
<p>As time passes, you’ll start to show up for specific keywords too — once Google and other SEs fully index your website, and based on third party links to your site, and other criteria (of which we know little).</p>
<aside class="quote no-group" data-username="nicolaevansfineart" data-post="4" data-topic="121028">
<div class="title">
<div class="quote-controls"></div>
 nicolaevansfineart:</div>
<blockquote>
<p>Anyway thanks for your help</p>
</blockquote>
</aside>
<p>You’r most welcome.</p>

**Comment by BaileyJM02** [comment]
<p>If you want to see all of the pages indexed for your website, you can search <code>site:&lt;domain&gt;</code>, for your domain it would be: <code>site:nicolaevansfineart.com</code>.</p>

**Comment by nicolaevansfineart** [comment]
<p>Thanks Bailey :heart::heart::+1::+1::grinning:</p>

**Comment by nicolaevansfineart** [comment]
<p>Hi,</p>
<p>The site is no longer showing on Google. Do you have any idea what I could try? Or where I could look for answers?</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="nicolaevansfineart" data-post="8" data-topic="121028">
<div class="title">
<div class="quote-controls"></div>
 nicolaevansfineart:</div>
<blockquote>
<p>The site is no longer showing on Google.</p>
</blockquote>
</aside>
<p>It still shows when Googling for the domain (both <a href="http://nicolaevansfineart.com" rel="noopener nofollow ugc">nicolaevansfineart.com</a> and <a href="http://www.nicolaevansfineart.co.uk" rel="noopener nofollow ugc">www.nicolaevansfineart.co.uk</a>) as well as via keywords searches like “NICOLA EVANS Fine Art”.</p>
<p>But the <a href="https://www.nicolaevansfineart.co.uk/" rel="noopener nofollow ugc">https://www.nicolaevansfineart.co.uk/</a> domain seems broken right now, even though it seems to have been indexed by Google at some point.</p>
<p>I’m not sure why you’re not seeing it in your searches, but it’s also true that Google results will differ based on which regional area you’re doing the search from.</p>

**Comment by nicolaevansfineart** [comment]
<p>I’ve just searched again and can see it showing as about the sixth result. The .<a href="http://co.uk" rel="nofollow noopener">co.uk</a> domain is an old site hosted by wix, the account with wix has been cancelled so I guess I need to ask them to remove that. I’ll have sometime over the weekend to dig in to what else I can do!</p>
<p>Thanks for your help mate, I really appreciate it!</p>

**Comment by ghost** [comment]
<p>Hi! To find out what I can do, say <code>@discobot display help</code>.</p>

**Comment by YankieGurl75** [comment]
<p>Help me make a certificate</p>

**Comment by YankieGurl75** [comment]
<p>How do make a an artificial intelligence</p>

**Comment by airtower-luna** [comment]
<p><a class="mention" href="/yankiegurl75">@YankieGurl75</a> This is a forum, not a search engine. :wink:</p>
<p>If you have a question that’s not direct related to the current thread, please create a new thread and give enough detail so people who might be able to answer can get an idea what you’re trying to figure out. The more specific the better.</p>

**Comment by samsalgado** [comment]
<p>Hey,<br>
I am having problems with setting up my custom domain via Godaddy. I did all the steps and called Godaddy twice and my site still does not show up online. Is it an issue with my package.json file??<br>
I recently changed the name of the homepage to the godaddy instead of the gh pages, is that the solution?<br>
Site name is <a href="http://themerlingroupworld.com" rel="noopener nofollow ugc">themerlingroupworld.com</a><br>
Repo: <a href="https://github.com/samsalgado/merlingroupworld" class="inline-onebox" rel="noopener nofollow ugc">GitHub - samsalgado/merlingroupworld</a></p>

**Comment by CharmRock** [comment]
is it possible to make user.github.io visible in google search? I mean free Github hosting? I posted the site a month ago and it still invisible for google. I have no own hosting and did actions in google Console. It says all is ok but site is still invisible. I worked only with "Resource with URL prefix" possibilities, because I have no rights on DNS. Is it impossible? Should I buy domain name to be visible in google only?

**Comment by nicolaevansfineart** [comment]
I've checked your repo and it doesn't look like you've got a robots.txt
file, this is used by search engine crawlers to index your site. I'd also
make sure you have a meta tag with some keywords in, also make sure that
site performs well on the Google lighthouse feature of the dev tools.

I did lots of googling to help me find out the issues so that could help
you too. Also make sure everything looks good in the Google console!
